### DISCUSSION TITLE
не запускается install_bin.sh

### DISCUSSION THREAD

**Discussion by lopatin531** [bol-van/zapret#805] [Q&A]
OpenWRT 22. Всё работало две недели назад.  Сегодня не запускает install_bin.sh и install_easy.sh. В первом случает пишет, что фала нет в директории (ls показывает, что есть). Мак и виндоус одинаково. На 25-30 попытке ввода одной и той же команды почему-то запустилось. Что может быть?

**Comment by bol-van** [comment]
Подробнее. Лог вводимых команд и результатов.
windows здесь причем ? install не работает на windows

**Comment by lopatin531** [comment]
Имел в виду, что вряд ли это зависит от маковских или виндоуских заморочек. 
Лог смогу завтра, уехал от комьютера.

**Reply by lopatin531** [reply]
скачал сегодня zip, залил в tmp по ssh, далее:

root@OpenWrt:~# cd /tmp
root@OpenWrt:/tmp# chmod -R 777 ./*
root@OpenWrt:/tmp# ./install_bin.sh
no compatible binaries found
root@OpenWrt:/tmp# ls
Makefile           etc                ipset              run
TZ                 extroot            lib                shm
blockcheck.sh      files              lock               spool
board.json         hosts              log                state
common             init.d             mdig               sysinfo
config.default     install_bin.sh     nfq                tmp
dhcp.leases        install_easy.sh    overlay            tpws
dnsmasq.d          install_prereq.sh  resolv.conf        uninstall_easy.sh
docs               ip2net             resolv.conf.d
root@OpenWrt:/tmp# ./install_easy.sh
* checking system
system is based on openwrt
package manager opkg
firewall fw4. flow offloading requires nftables.
* checking executables
find: /tmp/binaries: No such file or directory
build tools not found

press enter to continue


Я полный ноль, извиняюсь, сели допустил какую-то глупую ошибку

**Reply by bol-van** [reply]
Релиз качаем, а не исходники

**Reply by zingiev** [reply]
![image](https://github.com/user-attachments/assets/19d45761-a74e-48c2-ab39-87049248e522)
Скачал релиз сделал все по инструкции выходит ошибка

**Reply by fortoochka** [reply]
скачайте релиз с бинарниками.
https://github.com/bol-van/zapret/releases/download/v70.3/zapret-v70.3.zip

**Reply by KseniaSh2** [reply]
а поделитесь инструкцией, пожалуйста?

**Reply by HentayHentay** [reply]
Чтение талмута а не инструкция. Там все подробно написано 
Для винды есть отдельный гит
